.github/workflows/.github/workflows/ARC_llama_recipes_pytest_on_single_nvidia_gpu_node.yaml

```diff
@@ -0,0 +1,92 @@
+name: "[ARC][GPU] llama-recipes Pytest tests on Self-Hosted GPU k8s Nodes."
+on:
+  pull_request:
+    branches:    
+      - 'main'
+    paths:
+      - 'src/llama-recipes/configs/*.py'
+      - 'src/llama-recipes/utils/*.py'
+      - 'src/llama-recipes/*.py'
```

Comment:
wonder if we should cover `src/llama-recipes/data/*.py`  and `src/llama-recipes/dataset/*.py`as well? as we have set of tests for [datasets](https://github.com/facebookresearch/llama-recipes/tree/main/tests/datasets)?